base_user_role/views/user.xml

```diff
@@ -12,7 +12,7 @@
             <page string="Roles">
                 <field name="role_line_ids" nolabel="1">
                     <tree editable="bottom" colors="grey: not is_enabled;">
-                        <field name="role_id"/>
+                        <field name="role_id" required="1"/>
```

Comment:
Feel free to propose a PR to this PR to improve it if you wish. This PR was just a quick for to avoid the crash, nothing more.